provisioners/configure.sh

```diff
@@ -71,10 +77,6 @@ then
    update-alternatives --quiet --install /usr/bin/nodejs nodejs /usr/bin/node 50 --slave /usr/share/man/man1/nodejs.1.gz nodejs.1.gz /usr/share/man/man1/node.1.gz
 fi
 
-# Update nodejs to version 14
-npm install -g n
-n 14.18.3
```

Comment:
Note that this changes the approach taken in #121! The goal is to continue to get security updates for the 14.x line of Node.

If Permanent wants to move away from using the apt repo with [NodeSource's Node.js binary distributions](https://github.com/nodesource/distributions), that would not be an unreasonable step, but should be comprehensive - in that scenario, the `/etc/apt/sources.list.d/nodesource.sources` file should be removed, and another approach for installing the current minor version of the selected major version should be adopted throughout.